### PR TITLE
Fixes #35

### LINKED ISSUE #35: Need to clean up evolution-process per IP review
Our process documentation still references the microprofile.io site in several locations.  We need to clean these up per the IP review:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=12628

2.  The microprofile-evolution-process-master/process.md references
contributing to https://microprofile.io/community/#contributing.  I believe
you'll want to amend necessary URLs as required as this could be confusing.
<kws>Yes, agree.  We've been finding several of these ourselves and have been
correcting some of them as we go along.  We are also undergoing a massive
update to this "evolution process", so much of this might be removed.</kws>

URLS Generally

3.  It seems certain URLs remain pointing to the original project as above. 
Please advise if the content has received a pass to update URLs as expected?
<kws>Not sure what you mean by "receiving a pass"?  We will still have some
URLs pointing back at the microprofile.io site, but we shouldn't be relying or
depending on these URLs for any specific content.  It's more to recognize that
microprofile.io is a viable and useful portal for our efforts.</kws>

Third Party Content:

4.  The /microprofile-evolution-process-master/NOTICE.md indicates the work is
derived from the Swift Evolution Process and adapted to suit....  That
statement would indicate the inclusion of Third Party Content.  A Third Party
CQ is required.
<kws>Again, we're in the process of revamping our processes, so this particular
statement about Swift might go away.  Also, please be aware that this repo is
only to document our process for developing MicroProfile content.  This is not
part of our deliverables, per se.  These processes help us develop our
deliverables.  So, even this derived usage of the Swift Evolution Process is
not really Third Party Content.  There is nothing "real", just processes.</kws>

Since we're in the process of redefining and simplifying our process, we should clean these items up at the same time.

### PR DESCRIPTION
**Pull request by kenfinnigan**
Update to link to new process for feature ideas.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Eclipse MicroProfile
+
+This repository is no longer the active process for submitting feature ideas for Eclipse MicroProfile.
+
```
Comment (kwsutter) [inline README.adoc:4]: Ken, where would we keep the approved proposals?  We'd still want a central location to have all of our MP proposals available, wouldn't we?

Comment (kenfinnigan) [reply README.adoc:4]: Do we need to keep them somewhere?

Each approved proposal will have its own repository created for it that can act as the "source of truth".

Any proposal that isn't "approved", would either be present in the sandbox repository in some intermediate state, or wouldn't exist as no one has created a thread to discuss it.

If we need some central place, then I'd suggest that the Wiki have a link to the sandbox and all repos for each proposal. I wouldn't go beyond that

Comment (kwsutter) [reply README.adoc:4]: Fair enough.  Maybe that could be part of the process is to ensure that the Proposal is kept as part of their eventual repo.  Establishing a wiki page to keep track of the proposals and repos might be a nice reference.  I'm just thinking that we don't want prospective users to have to know where to look (or hunt) to track down this information.

**Review by kwsutter — APPROVED** [review]
Looks good.

**Pull request merged**